src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -594,10 +593,15 @@ public VisualEditorPageObject openNewArticleEditModeVisual(String wikiURL) {
     return new VisualEditorPageObject(driver);
   }
 
+  public String getEditToken() {
+    return (String) jsActions.execute("mw.user.tokens.get('editToken')");
+  }
+
   public void addVideoViaAjax(String videoURL) {
-    jsActions.execute(
-        "$.ajax('" + getWikiUrl() + "wikia.php?controller=Videos&method=addVideo&format=json', {"
-            + "data: {url: '" + videoURL + "'}," + "type: 'POST' } );");
+    String editToken = getEditToken();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "editToken" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)